package.json

```diff
@@ -26,7 +26,7 @@
     "html-webpack-plugin": "^5.5.0",
     "loadash": "^1.0.0",
     "style-loader": "^3.3.1",
-    "webpack": "^5.70.0",
+    "webpack": "^5.71.0",
     "webpack-cli": "^4.9.2",
     "webpack-dev-server": "^4.7.4"
   },
```

Comment:
- [ ] Please, on top of your `package.json` file, on your `scripts`, do something like `"start": "webpack --config webpack.config.js --watch"` so this way you can run `npm start` on your terminal to see your page working.

Reply:
Thank you so much for the help! I have made the requested changes.